reconcile/gitlab_labeler.py

```diff
@@ -88,8 +91,10 @@ def guess_labels(project_labels: Iterable[str],
         path_dir = os.path.dirname(path)
         path_tokens = path_dir.split('/')
 
-        tenants += [t for t in path_tokens if t not in ignore_tokens and
-                    t in apps]
+        if 'data/services/' in path:
+            tenants += [t for t in path_tokens if t not in ignore_tokens and
+                        t in apps]
+
```

Comment:
Although I would love if we could find more heuristics to find labels, this should be enough for now, at least it will be real.